runner/ansible/roles/2.1.1/tasks/main.yml~

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This file looks some kind of old backup. We should remove it, right?

Reply:
yep

Reply:
removed